connectors/src/resources/storage/models/connector_model.ts

```diff
@@ -72,10 +71,6 @@ ConnectorModel.init(
       type: DataTypes.STRING,
       allowNull: false,
     },
-    dataSourceName: {
```

Comment:
so the plan is to wait a little before running the column deletion so we can rollback if needed correct?

Reply:
Exactly